src/App.tsx

```diff
@@ -21,7 +21,7 @@ export default function App() {
   // vvvvvvvvvvvvvvvvv
 
   let yourBlock: BlockModel = {
-    type: "%NAME%",
+    type: "FlashingText",
```

Comment:
I've been meaning to write a git commit hook to automatically return this to default, but for now, could you place it back to `"%NAME%"`? This will allow the `seam-magic` script to run for the next person.